spec/integration/worker_async_spec.rb

```diff
@@ -14,7 +12,8 @@ module Backend
       subject { Worker.new(async: true, job_fetcher: job_fetcher) }
 
       let(:storage) { Service.storage }
-      let(:resque_redis ) { job_fetcher.instance_variable_get(:@redis) }
+      let(:redis_client ) { job_fetcher.instance_variable_get(:@redis).instance_variable_get(:@redis)
+                                       .instance_variable_get(:@redis).instance_variable_get(:@redis_async) }
```

Comment:
I think it's much better if we use `::Resque.redis` and add `#blmove` to the async client. If ever we start relying on this method in sync client as well, we should write tests that validate same behavior.

I think the mess with getting instance variables is bigger than potential method behavior difference.